lib/augmentConfig.mjs

```diff
@@ -3,6 +3,7 @@ import { dirname, isAbsolute } from 'node:path';
 import globjoin from 'globjoin';
 import micromatch from 'micromatch';
 import normalizePath from 'normalize-path';
+import process from 'process';
```

Comment:
nit

`'node:process'` for consistency

Reply:
Agree. I want to distinguish the standard library and 3rd-party libraries clearly in imports.